pydarn/io/dmap.py

```diff
@@ -62,7 +62,7 @@
 # Dictionary of DMAP types (key) to quickly convert the format and byte size
 # (value-tuple)
 DMAP_DATA_TYPES = {DMAP: ('', 0),
```

Comment:
It would be helpful to have comments next to 'dmap' and 'short' as well.

Reply:
Done, also commented the rest of the DMap types.